test/helper/test_timeline.ts

```diff
@@ -339,6 +575,13 @@ const testTimelineFiles = (timelineFiles: string[]): void => {
             }
           }
         });
+        it('should not have lint', () => {
+          let errorStr = '';
```

Comment:
Minor nit, ignore as desired, but this might be better phrased as "should not have lint errors".